autodepgraph/graph.py

```diff
@@ -302,6 +317,44 @@ def draw_svg(self, filename: str=None):
         self._update_drawing_attrs()
         vis.draw_graph_svg(self, filename)
 
+    def open_html_viewer(self):
+        """ Open html viewer for the file specified by the svg backend """
+        template = os.path.join(os.path.split(autodepgraph.__file__)[0], 'svg_viewer', 'svg_graph_viewer.html')
+        with open(template, 'rt') as fid:
+            x=fid.read()
+            
+        base, file = os.path.split(self.cfg_svg_filename)        
+        x=x.replace('adg_graph.svg', file)
+        
+        tfile=tempfile.mktemp(prefix='svgviewer-', suffix='html', dir=base)
+        with open(tfile, 'wt') as fid:
+            fid.write(x)
+        webbrowser.open_new_tab(tfile)
+        return tfile
+
+    def set_node_attribute(self, node, attribute, value):
+        """ Set the attribute of the specified node 
+        
+        Args:
+            node (str): name of the node
+            attribute (str): attribute to set
+            value (ojbect): value to set
+        """
+        nx.set_node_attributes(self, {node: {attribute: value}})
+
+    def set_node_description(self, node, description):
+        """ Set the node description field
+        
+        Args:
+            node (str): name of the node
+            description (str): description to set
+        """
+        nx.set_node_attributes(self, {node: {'description': description}})
+
+    def calibration_state(self):
```

Comment:
@AdriaanRol It is shorter and it is part of the graph object (I think you are talking about `set_node_attribute`). Then a user can enter (in spyder or jupyter notebook):
```
mygraph. [tab]
```
and get a list of possible methods. How would a normal user find out that there is a method in the `nx` module to modify the attributes?

Reply:
About the `calibration_state`: this makes sense from a user perspective. Right now this is just `graph.node`, but maybe later it will include other information as well. Also I want a `dict` to be returned, since I use it in a `qcodes.Parameter` `get_cmd`.